### PR TITLE
Added snackbar, story book

### PR DESCRIPTION
**Pull request by elayabharath**
- Added snackbar UI element.
- Setup css with variables [not sure if this is a good idea].
- Setup a storybook to show example of the element.

_Upcoming next_
Integrate snackbar to all repos

### CODE REVIEW

**Review by marianamarasoiu — COMMENTED** [review]
Hi @elayabharath this is really great, thanks! first round of comments, mostly very minor.

Really great idea to use CSS variables - I've been thinking we should use them for a while, so great to use this opportunity to introduce them.

Also great to have a "storybook" to test these components out. What do you think about adding a button to the storybook that shows the snackbar when clicked?

On another front, I think we should also treat the CSS for the components themselves in the same way we treat the dart files, so based on https://dart.dev/tools/pub/package-layout it seems that the guidelines suggest that we can have css files alongside dart files in `lib/`. Given that we expect to have a few other css files that don't have a dart counterpart, I'm thinking it may be a good idea to keep the css files in `lib/css` instead rather than directly under `lib/`? Then any other css files that are styling the storyboard itself can live under `web/`

File: lib/snack_bar.dart [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    _contents = new DivElement()..classes.add('contents');
+    snackbarElement.append(_contents);
+  }
+
+  showSnackbar(String message, SnackbarNotificationType type) {
+    _contents.text = message;
+    snackbarElement.classes.remove('hidden');
+    snackbarElement.setAttribute(
+        'type', type.toString().replaceAll('SnackbarNotificationType.', ''));
```
[the remark this answers is not in the document]
Comment (elayabharath) [reply lib/snack_bar.dart]: Have set it to 120, along with other settings using .vscode > settings.json and committed in here

**Comment by elayabharath** [comment]
Fixed all the issues 
- Renamed `snack_bar` to `snackbar`
- settings.json / notably adds the new line at the end of files, 120 line width for folding
- moved css to under `/lib`
- Removed the config option on the snackbar time on screen

Could you take a look again? Thanks! @marianamarasoiu

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        "**/Icon?": true
+    },
+    "files.trimFinalNewlines": true,
+    "files.trimTrailingWhitespace": true,
+    "files.insertFinalNewline": true,
+    "html.format.endWithNewline": true,
+    "dart.lineLength": 120,
+    "[dart]": {
+        "editor.rulers": [
+            120
```
Comment (marianamarasoiu) [inline .vscode/settings.json:14]: I just had a chat with Dan and we were discussing 160 for the backend code - I'd be inclined to revisit our 120 and use the same number here as well? what do you think?
```suggestion
            160
```

File: web/script.dart [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+DivElement snackbarContainer = querySelector('#snackbar-container');
+ButtonElement snackbarTrigger = querySelector('#show-snackbar');
+
+void main() {
+  // snackbar
+  SnackbarView snackbarView = SnackbarView();
+  snackbarContainer.append(snackbarView.snackbarElement);
+  snackbarTrigger.onClick.listen((_) {
+    snackbarView.showSnackbar("Welcome to KatiKati UI library!", SnackbarNotificationType.success);
```
Comment (marianamarasoiu) [inline web/script.dart:12]: nit:
```suggestion
    snackbarView.showSnackbar("Welcome to the Katikati UI library!", SnackbarNotificationType.success);
```

**Review by marianamarasoiu — APPROVED** [review]
thanks EB, LGTM with just a couple of very small comments!

**Pull request merged**